src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -9,13 +13,25 @@
 public class SelectTestFigureOptionListener implements ActionListener {
 
 	private DriverManager driverManager;
+	private String figure;
 
-	public SelectTestFigureOptionListener(DriverManager driverManager) {
+	public SelectTestFigureOptionListener(DriverManager driverManager, String name) {
 		this.driverManager = driverManager;
+		this.figure = name;
 	}
 
 	@Override
 	public void actionPerformed(ActionEvent e) {
-		FiguresJoe.figureScript1(driverManager.getCurrentDriver());
+		if (figure == "figureScript1")
+			FiguresJoe.figureScript1(driverManager.getCurrentDriver());
+		else if (figure == "figureScript2")
+			FiguresJoe.figureScript2(driverManager.getCurrentDriver());
+		else if (figure == "rect"){
```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.